src/Utils/Logger.ts

```diff
@@ -34,27 +36,27 @@ export class Logger {
 
     return Logger.logger;
   }
-
-  private checkShow() {
-    if (this.firstFocus) {
-      this.outputChannel.show(false);
-      this.firstFocus = false;
+*/
+  private static checkShow() {
+    if (Logger.firstFocus) {
+      Logger.outputChannel.show(false);
+      Logger.firstFocus = false;
     }
   }
 
-  public outputWithTime(msg: string) {
+  public static outputWithTime(msg: string) {
     let dateTime = new Date();
-    this.checkShow();
-    this.outputChannel.appendLine('[' + dateTime.toLocaleString() + '] ' + msg);
+    Logger.checkShow();
+    Logger.outputChannel.appendLine('[' + dateTime.toLocaleString() + '] ' + msg);
   }
 
-  public output(msg: string) {
-    this.checkShow();
-    this.outputChannel.append(msg);
+  public static output(msg: string) {
```

Comment:
(Q) What about `Logger.log()` or `Logger.logLine` `Logger.logWithTime`? For (1) It sounds better with verb at the front and (2) `log` sounds familiar than `output` (ex. console."log"). BTW I understand that Logger logs just like console.log but into its